srcpkgs/gnome-system-monitor/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'gnome-system-monitor'
 pkgname=gnome-system-monitor
-version=41.0
+version=42.0
 revision=1
 build_style=meson
 configure_args="-Dsystemd=false"
-hostmakedepends="pkg-config intltool itstool glib-devel polkit"
+hostmakedepends="pkg-config intltool itstool glib-devel polkit gtk4-update-icon-cache"
```

Comment:
```suggestion
hostmakedepends="pkg-config gettext itstool glib-devel polkit"
```

Reply:
`gtk4-update-icon-cache` is necessary because of the meson update.

Reply:
Thank you.